app/views/users/_profile.html.erb

```diff
@@ -43,158 +35,30 @@
-    </div>
-  </div>
-
-  <div class="crayons-card mb-6 grid gap-6 p-6">
-    <h2>Links</h2>
-
-    <div class="crayons-field">
-      <%= f.label :facebook_url, "Facebook profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :facebook_url, class: "crayons-textfield", placeholder: "https://facebook.com/..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :youtube_url, "Youtube URL", class: "crayons-field__label" %>
-      <%= f.url_field :youtube_url, class: "crayons-textfield", placeholder: "https://www.youtube.com/channel/..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :stackoverflow_url, "Stack Overflow profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :stackoverflow_url, class: "crayons-textfield", placeholder: "https://stackoverflow.com/users/..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :linkedin_url, "LinkedIn profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :linkedin_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :behance_url, "Behance profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :behance_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :dribbble_url, "Dribbble profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :dribbble_url, class: "crayons-textfield", placeholder: "https://dribbble.com/..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :medium_url, "Medium profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :medium_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :gitlab_url, "GitLab profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :gitlab_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :instagram_url, "Instagram profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :instagram_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :mastodon_url, "Mastodon profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :mastodon_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-
-    <div class="crayons-field">
-      <%= f.label :twitch_url, "Twitch profile URL", class: "crayons-field__label" %>
-      <%= f.url_field :twitch_url, class: "crayons-textfield", placeholder: "https://..." %>
-    </div>
-  </div>
+  <% @profile_field_groups.each do |group| %>
+    <% unless group.profile_fields.empty? %>
+      <div class="crayons-card mb-6 grid grid-flow-row gap-6 p-6">
+        <h2><%= group.name %></h2>
+        <% if group.description %>
```

Comment:
I think we better use `group.description.present?` here since empty strings are truthy in Ruby.

Reply:
✅